src/components/widgets/Compass.vue

```diff
@@ -80,6 +80,12 @@ onBeforeMount(() => {
   }
 })
 
+onMounted(() => {
+  // Set initial value to 0.01 since 0.0 and 360 does not render anything
+  adjustLinesX(0.01)
```

Comment:
Did you try setting yaw ref to 0.1 ? As you are doing for roll and pitch ?

Reply:
Moved to use ref(0.01) instead of parameter in adjustLineX